packages/@aws-cdk/cloudformation-diff/lib/diff/types.ts

```diff
@@ -117,6 +120,22 @@ export class DifferenceCollection<V, T extends Difference<V>> {
     return Object.keys(this.changes);
   }
 
+  /**
+   * Removes all changes that do not match the specified filter.
+   */
+  public applyFilter(filter: (diff: T | undefined) => boolean) {
```

Comment:
I so would have preferred for this to return a new `TemplateDiff` object.

Reply:
Me too.

Reply:
You got it. Fixed.